_posts/2025-06-24-kafka-queue.md

```diff
@@ -0,0 +1,703 @@
+---
+layout: post
+title:  "Using Queues for Kafka with Strimzi"
```

Comment:
I think it would be nice to use the full name in the title and in the first sentences ... I know the KIP is named _Queues for Kafka_ ... but still.

```suggestion
title:  "Using Queues for Apache Kafka with Strimzi"
```